grpc-codegen/src/main/scala/monix/grpc/codegen/package.scala

```diff
@@ -0,0 +1,43 @@
+package monix.grpc
+
+import protocbridge.{Artifact, SandboxedJvmGenerator}
+import scalapb.GeneratorOption
+import scalapb.GeneratorOption.FlatPackage
+
+/**
+  * Copyright (C) 08.11.21 - REstore NV
+  */
+
+package object codegen {
```

Comment:
Why a package object?

Reply:
It is a copy of the codegen package object in scalapb. I could make it a lower cased name object.